.dockerignore

```diff
@@ -18,5 +18,18 @@ kubeconfig
 **/mlrun.env
 mlrun.egg-info
 
+**/*.coverage
+**/.aws-secrets
+**/credentials.txt
+**/credentials.json
+**/test-alibaba-oss.yml
+**/test-azure-blob.yml
+**/test-aws-s3.yml
+**/test-google-cloud-storage.yml
+**/test-dbfs-store.yml
+
+**/patch-env.yml
```

Comment:
can join the two entries to `**/patch[-_]env.yml`

Reply:
changed to **/patch[_\-]env.yml because "-" is  a special character.